titus-common/src/main/java/com/netflix/titus/common/util/DateTimeExt.java

```diff
@@ -147,4 +148,11 @@ public static String toRateString(long interval, long itemsPerInterval, TimeUnit
 
         return String.format("%.2f %s/%s", rate, rateType, toTimeUnitAbbreviation(matchedTimeUnit));
     }
+
+    /**
+     * Get the {@link OffsetDateTime} representation from specified epoch in milliseconds.
+     */
+    public static OffsetDateTime fromMillis(long epochMillis) {
```

Comment:
is this being used in other external projects as part of the `titus-common` library?